src/zenml/models/model_models.py

```diff
@@ -237,6 +237,9 @@ def to_model_version(
             was_created_in_this_run=was_created_in_this_run,
             suppress_class_validation_warnings=suppress_class_validation_warnings,
         )
+        mv._id = self.id
+
+        return mv
```

Comment:
@avishniakov I discovered this value was never set, i implemented a temporary fix but would leave it up to you to decide where to handle this